Resources/doc/index.rst

```diff
@@ -124,11 +124,18 @@ An example builder class would look like this:
     // src/AppBundle/Menu/Builder.php
     namespace AppBundle\Menu;
 
+    use Doctrine\Common\Persistence\ManagerRegistry;
     use Knp\Menu\FactoryInterface;
-    use Symfony\Component\DependencyInjection\ContainerAware;
 
-    class Builder extends ContainerAware
+    class Builder
     {
+        private $registry;
+        
+        public function __construct(ManagerRegistry $registry)
```

Comment:
This is wrong. This example is about documenting menus rendered as ``knp_menu_render('AppBundle:Builder:mainMenu')``, so you cannot inject the ManagerRegistry in the constructor. This should be using ContainerAwareInterface here.

Registering builders as services is documented in a different chapter